tools/check-git-version.js

```diff
@@ -0,0 +1,21 @@
+import semver from 'semver';
+import shell from 'shelljs';
+import simpleGit from 'simple-git';
+
+const GIT_MINIMUM_VERSION = '2.33.0';
+const git = simpleGit();
+async function checkGitVersion() {
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/no-floating-promises
(async () => {
```

Use simple arrow function with wrapper